### PR TITLE
Upgrade the search mechanism to cards with a YES\NO interface

### LINKED ISSUE #179: Upgrade the way filtered apartments shown to Yes\No interface
**Is your feature request related to a problem? Please describe.**
Right now in the search mechanism, in the 'results.html' file the filtered apartments are shown poorly.

image

**Describe the solution you'd like**
The filtered apartments should be shown with a Yes\No interface.
In the next PR's the 'results.html' file should be upgraded.

### PR DESCRIPTION
**Pull request by AmitAharoni**
Upgrade the way filtered apartments shown to cards.
Upgrade the way filtered apartments shown to Yes\No interface.

# Description
Upgrade the way filtered apartments shown to cards.
Upgrade the way filtered apartments shown to YES\NO interface.

## Screenshots
image

### Issues closed by this PR
fix: #179, #180.

### PR Dependencies
NONE.

### CODE REVIEW

**Comment by Micha546** [comment]
Are you showing 1 reasult at a time? or is there a way to see all the cards the search found?

File: search/templates/search/search.html [hunk 1]
```diff
@@ -6,19 +6,98 @@
...
+<div class="row">
+    <div class="col-md-5"> 
+        <h2>Hey {{ loggedUser.first_name }}!</h1>
+        <h4>Search for apartments now!</h2>
+        <form method="POST">
+            {% csrf_token %}
+            <fieldset class="form-group">
+                {{ searchForm|crispy }}
+                <button type="submit">Search!</button>
+            </fieldset>
```
Comment (tamirho) [inline search/templates/search/search.html:18]: I recommend you to add some CSS boostrap classes to the button.
Try this:
`<button class="btn btn-primary mt-3" type="submit">{your text}</button>`

`btn btn-primary` stands for the button style.
`mt-3` stands for `margin-top 3px`.

**Review by tamirho — COMMENTED** [review]
> Are you showing 1 reasult at a time? or is there a way to see all the cards the search found?

Can you please attach a full screenshot so we can see what it looks like?

File: search/templates/search/search.html [hunk 2]
```diff
@@ -6,19 +6,98 @@
 {% endblock %}
 
 {% block content %}
-<div>
-    <h1>Hey {{ loggedUser.first_name }}!</h1>
```
Comment (DanielMy777) [inline search/templates/search/search.html:10]: You dont have to send 'loggedUser' in context,
django will automatically send the request user under the name 'user'.
for example this line can be switched to `<h1>Hey {{ user.first_name }}!</h1>` without sending special context.

**Comment by DanielMy777** [comment]
> Are you showing 1 reasult at a time? or is there a way to see all the cards the search found?

He's using a for loop so its not one at a time, I wonder what it looks like tho.
@AmitAharoni can you send us a screenshot?

**Comment by AmitAharoni** [comment]
> Are you showing 1 reasult at a time? or is there a way to see all the cards the search found?
@Micha546 
Showing all cards. Moving via the scroll down.

File: search/templates/search/search.html (hunk 1 above)
Comment (AmitAharoni) [reply search/templates/search/search.html:18]: I'll try 
@tamirho

**Review by Micha546 — APPROVED** [review]

**Review by DanielMy777 — APPROVED** [review]
LGTM
inspect the note I wrote, it's not that important but a good time saver.

**Pull request merged**